tests/unit/acl.tcl

```diff
@@ -145,6 +145,7 @@ start_server {tags {"acl"}} {
         $rd AUTH psuser pspass
         $rd CLIENT SETNAME deathrow
         $rd SUBSCRIBE foo:1
+        $rd read
```

Comment:
that's just reading the AUTH response. i suppose you need 2 more `read`s?

Reply:
👍